en/sensor/sf1xx_lidar_setup.md

```diff
@@ -26,3 +27,18 @@ Use the `SENS_EN_SF1XX` parameter to select the lidar model and then reboot.
 * `2` SF10/b
 * `3` SF10/c
 * `4` SF11/c
+
+For PX4v2 you also need to add it to the make file:
+open up /cmake/config/nuttx_px4fmu-v2_default.cmake
+
+You should see a list of drivers.
+Make sure /drivers/sf1xx is not commented out (no # sign in from of that line)
+
+If you are using a pixhawk2, you will also have to edit the driver. 
```

Comment:
This is OK for now, but we will be be fixing the properly in the codebase shortly. Please mark this as a temporary hack for Pixhawk 2.1.